app/frontend/shared/components/ErrorState.res

```diff
@@ -1,19 +1,27 @@
+@module("../images/page-not-found.svg")
+external pageNotFoundSVG: string = "default"
+
 let str = React.string
 
 let defaultCTA = {
-  <p className="mt-2 mb-3 text-sm leading-normal"> {str("Try heading back?")} </p>
+  <button
+    onClick={_ => RescriptReactRouter.push("/")}
```

Comment:
@irajsuhail 
Back to home button is redirecting to root URL and getting an empty page.
Can we try this?
`onClick={_ => RescriptReactRouter.push("/school/courses")}`

Reply:
@vinutv  Yes, we can.

I am curious about what will happen if we use this error state in the student's view. Where will `/school/courses` navigate? 🤔

cc @bodhish

Reply:
Few changes, 
1. This should be a link, not a button. (Use <Link/>)
2. For now keep the back to `/school`. (If we are using it on the student's side, we can pass in the `cta` prop that will replace `defaultCTA`